src/model/test/FloorspaceReverseTranslator_GTest.cpp

```diff
@@ -82,6 +82,35 @@ using namespace openstudio::model;
     OutputDebugString(os_.str().c_str()); \
   }
 
+void CompareSurfaceGroups(Model& model, Model& baseline) {
+
+  auto shadingGroup1 = model.getModelObjects<ShadingSurfaceGroup>();
+  auto shadingGroup2 = baseline.getModelObjects<ShadingSurfaceGroup>();
+  EXPECT_EQ(shadingGroup1.size(), shadingGroup2.size());
```

Comment:
Before fix, the `FloorspaceReverseTranslator_shadingtest` reports a failure here, is that expected?

```
2524: Expected equality of these values:
2524:   shadingGroup1.size()
2524:     Which is: 12
2524:   shadingGroup2.size()
2524:     Which is: 11
```

`FloorspaceReverseTranslator_Issue_4764` reports equality. Both reports lots of failure on line 108 where matched is false.

Reply:
If it is, then I'm all good with this PR!

Reply:
@ggartside this is the last item that needs addressing before merging please. Is this expected or not?

Reply:
I ran the test here and it passes and it looks like all the tests pass. The shading groups should match.